src/test/java/dog/pawbook/storage/JsonAddressBookStorageTest.java

```diff
@@ -73,7 +77,11 @@ public void readAndSaveAddressBook_allInOrder_success() throws Exception {
 
         // Modify data, overwrite exiting file, and read back
         original.addEntity(HOON);
-        original.removeEntity(ALICE);
+        List<Pair<Integer, Entity>> targets = original.getEntityList().stream()
+                .filter(p -> p.getValue().isSameEntity(ALICE))
+                .collect(toList());
+        int aliceId = targets.get(0).getKey();
+        original.removeEntity(aliceId);
```

Comment:
same the other one

Reply:
I think for this I didn't use the asObservableList method